pymc/model_graph.py

```diff
@@ -439,6 +442,7 @@ def make_graph(
     figsize=None,
     dpi=300,
     node_formatters: NodeTypeFormatterMapping | None = None,
+    graph_attrs: GraphAttrMapping | None = None,
```

Comment:
Not sure I like the custom type, it makes it seem more complex than it really is. It's not much shorter than `dict[str, Any]`

Reply:
I have no strong feelings about this other than it was to be more consistent with the rest of the file

Reply:
up to you; not a blocker